src/cli.ts

```diff
@@ -283,33 +286,53 @@ yargs.command(
         try {
             const adapter = createAdapterFromUrl(argv.db!)
             await adapter.init()
-            const head = await getHead()
-            const status = await getStatus(adapter, head)
-            process.stdout.write(status.toString())
-            if (status.newCommits.some(commit => commit.tasks.length > 0)) {
-                if (argv.confirm) {
-                    const answer = await inquirer.prompt<{ continue: boolean }>({
-                        type: 'confirm',
-                        name: 'continue',
-                        message: 'Continue?',
-                    })
-                    if (!answer.continue) {
-                        process.exit(0)
+            while (true) {
```

Comment:
Should this retry an infinite amount of times?

Reply:
hmm do you think it should fail?